lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/auth/DBOAuthenticationDAOImpl.java

```diff
@@ -111,7 +108,11 @@
 			" WHERE "+COL_SESSION_TOKEN_SESSION_TOKEN+"= ?";
 	
 	private static final String SELECT_PRINCIPAL_BY_TOKEN_IF_VALID = 
-			SELECT_PRINCIPAL_BY_TOKEN+IF_VALID_SUFFIX;
+			"SELECT st."+COL_SESSION_TOKEN_PRINCIPAL_ID+
+			" FROM "+TABLE_SESSION_TOKEN+" st, "+TABLE_AUTHENTICATED_ON+" ao "+
+			" WHERE "+COL_SESSION_TOKEN_SESSION_TOKEN+"= ?"+
+			" AND ao."+COL_AUTHENTICATED_ON_PRINCIPAL_ID+"="+"st."+COL_SESSION_TOKEN_PRINCIPAL_ID+
```

Comment:
Maybe let us to the explicit join, e.g. FROM TABLE_SESSION_TOKEN+" st JOIN "+TABLE_AUTHENTICATED_ON+" ao ON ao."+COL_AUTHENTICATED_ON_PRINCIPAL_ID+"="+"st."+COL_SESSION_TOKEN_PRINCIPAL_ID